tests/logs/combined_push/test_combined_push/test_do_advisory_push.txt

```diff
@@ -102,7 +151,7 @@
 [   DEBUG] Searching for product sample_product for provider awstest in rhsm
 [   DEBUG] Searching for product sample_product_HOURLY for provider awstest in rhsm
 [   DEBUG] Searching for product sample_product for provider awstest in rhsm
-[    INFO] Uploading /foo/bar/sample_product_test.raw to region us-east-1 (type: hourly, ship: True)
+[    INFO] Uploading /foo/bar/sample_product_test.raw to region us-east-1 (type: hourly, ship: True) with sharing accounts: ['0', '0', '0', '0', '0', '0', '1', '2', '1', '2', '1', '2'] and snapshot accounts: ['1', '1', '1', '1', '1', '1', '1', '2', '1', '2', '1', '2']
```

Comment:
@JAVGan list of characters is a bit weird in the logs; is there something wrong in mocked data?

Reply:
I'll check it

Reply:
Nice catch!

The mock data was using a wrong format.

FIX: https://github.com/release-engineering/pubtools-marketplacesvm/pull/40